src/sql/platform/query/common/queryRunner.ts

```diff
@@ -404,13 +408,12 @@ export default class QueryRunner extends Disposable {
 	/**
 	 * Handle a Mssage from the service layer
 	 */
-	public handleMessage(obj: azdata.QueryExecuteMessageParams): void {
-		let message = obj.message;
-		message.time = new Date(message.time!).toLocaleTimeString();
-		this._messages.push(message);
+	public handleMessage(messagesObj: azdata.QueryExecuteMessageParams[]): void {
+		const messages = messagesObj.map(m => m.message);
+		this._messages.push(...messages);
```

Comment:
this is a behavior change to not format the time string until we have to, @chlafreniere this is a behavior change that probably affects notebooks.

Reply:
Thanks, I'll take a look. The ordering of messages should be consistent I'd imagine though, right?

Reply:
Yes order is still maintained.